### PR TITLE
Added new report option

### PR DESCRIPTION
**Pull request by theshanx**
This commit adds an export flag to enagle GADS tooling.

### CODE REVIEW

**Comment by kylog** [comment]
Hey @stahnma: in your copious spare time, are you interested in helping out with, ahem, maintenance of this maintainers gem?

File: lib/maintainers/runner.rb [hunk 1]
```diff
@@ -271,7 +289,9 @@ def report(options)
       when 'repo'
         report_repo_details(maintainers_files)
       when 'people'
-        report_people_details(maintainers_files)
```
Comment (MikaelSmith) [inline lib/maintainers/runner.rb:274]: I'm curious what changed on this line.

File: lib/maintainers/cli.rb [hunk 2]
```diff
@@ -110,7 +110,7 @@ def parse(args)
             opts.on("-v", "--verbose", "verbosity") do |v|
               options[:verbose] = v
             end
-            opts.on("-d", "--details [repos|people]", "detailed report") do |v|
+            opts.on("-d", "--details [repos|people|lists]", "detailed report") do |v|
```
Comment (MikaelSmith) [inline lib/maintainers/cli.rb:114]: We should update the help at line 56 (I was a little surprised it's not generated from opts).

**Review by MikaelSmith — CHANGES_REQUESTED** [review]

File: lib/maintainers/runner.rb (hunk 1 above)
Comment (theshanx) [reply lib/maintainers/runner.rb:274]: This is not the first time git has highlighted a non-existent diff (and I've never really dug into root causes for it)

File: lib/maintainers/cli.rb (hunk 2 above)
Comment (theshanx) [reply lib/maintainers/cli.rb:114]: updated!

**Review by MikaelSmith — APPROVED** [review]

**Pull request merged**